src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -171,6 +215,35 @@ void RenderTarget::clear(const Color& color)
 }
 
 
+////////////////////////////////////////////////////////////
+void RenderTarget::clear(int value)
+{
+    if (RenderTargetImpl::isActive(m_id) || setActive(true))
+    {
+        // Unbind texture to fix RenderTexture preventing clear
+        applyTexture(nullptr);
+
+        glCheck(glClearStencil(value));
+        glCheck(glClear(GL_STENCIL_BUFFER_BIT));
```

Comment:
It looks to me the code related to applying the view for scissor testing we can see in `RenderTarget::clear(const Color&)` method above is missing here.
Same remark for the other `clear` method below.